### DISCUSSION TITLE
How to always collapse a category?

### DISCUSSION THREAD

**Discussion by tooomm** [release-drafter/release-drafter#1325] [Q&A] [answered]
`collapse-after: 0` seems to not work.

Looking at the implementing PR (https://github.com/release-drafter/release-drafter/pull/1095/files), `0` is the minimum allowed value and even the default according to "schema.json".
But also the code checks that `category['collapse-after'] !== 0`.

**Comment by jetersen** [comment, marked as answer]
`collapse-after: 1` is considered always collapse. If the is only one item there is no reason to collapse.

**Reply by tooomm** [reply]
Thanks for the clarification, @jetersen!
Would it make sense then to adjust the allowed minimum and default value to 1 in the [schema.json](https://github.com/release-drafter/release-drafter/blob/master/schema.json#L219-L221) as well?

We had `collapse-after: 0` in our code for weeks without realizing what was wrong as no error or hint was thrown.

**Reply by jetersen** [reply]
The default is zero for a reason that is the default value to turn off the feature.
